web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -70,46 +70,57 @@ const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
       <div className="d-flex togglers-wrapper">
         {stateColorTuples.map(([state, color]) => {
           return (
-            <Checkbox wrapperStyles={{ marginRight: 10 }} defaultChecked id={`${state}-toggler`} onClick={toggle(state)}>
+            <Checkbox
+              key={color}
+              wrapperStyles={{ marginRight: 10 }}
+              defaultChecked
+              id={`${state}-toggler`}
+              onClick={toggleFilter(state)}
+            >
               <Badge color={color} className="text-capitalize">
                 {state} ({statsCount[state]})
               </Badge>
             </Checkbox>
           );
         })}
-        <Checkbox
-          wrapperStyles={{ marginLeft: 'auto' }}
-          id="show-annotations-toggler"
-          onClick={() => setShowAnnotations(!showAnnotations)}
-        >
-          <span style={{ fontSize: '0.9rem', lineHeight: 1.9 }}>Show annotations</span>
-        </Checkbox>
+        {/* hide if all filters are off */}
```

Comment:
I would always show it - I don't see the association of this checkbox to the filter state of the groups, it's more of a global page setting...

Reply:
The association is not with the filters but with the fact that once nothing is left on the screen this checkbox becomes unusable. The other option is to disable it but if you prefer I'll leave it without any modifications

Reply:
You mean it doesn't affect anything that's currently visible (which seems fine to me), or it actually becomes broken somehow in that sense that it can't be clicked properly?

Reply:
Nothing breaks, just become unusable and something unusable makes more sense to me to be prevented from usage at all. Hiding it wasn't the best option yes :) but at least to disable it :)

Reply:
:man_shrugging: I dunno. I'd be for still just showing it TBH.

Reply:
Yeah, ok :)